pkg/scheduler/framework/plugins/feature/feature.go

```diff
@@ -27,4 +27,5 @@ type Features struct {
 	EnableInPlacePodVerticalScaling              bool
 	EnableSidecarContainers                      bool
 	EnableSchedulingQueueHint                    bool
+	EnableStorageCapacityScoring                 bool
```

Comment:
I don't see where is this field set based on `StorageCapacityScoring` feature gate

Reply:
I have set `EnableStorageCapacityScoring` in pkg/scheduler/framework/plugins/registry.go.